StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement.swift

```diff
@@ -104,6 +104,11 @@ public final class EmbeddedPaymentElement {
     public func update(
         intentConfiguration: IntentConfiguration
     ) async -> UpdateResult {
+        isUpdating = true
+        // Ensure `isUpdating` is reset by using a defer block, preventing accidental omissions in early returns.
+        defer {
+            isUpdating = false
+        }
```

Comment:
Does this work?

If you submit two updates U1 and U2 in a row, when U1 finishes it sets `isUpdating = false`.  What if U2 is still in progress?